.circleci/config.yml

```diff
@@ -69,28 +95,21 @@ jobs:
         type: enum
         enum: ['x64', 'arm64']
     executor: << parameters.executor >>
+    parallelism: 2
     steps:
       - install
       - when:
           condition:
             equal: [node/windows, << parameters.executor >>]
           steps:
-            - node/install-packages
             - run:
-                name: setup Windows and run slow tests
-                shell: powershell.exe -ExecutionPolicy Bypass
+                name: Windows Setup
+                shell: bash
                 command: |
-                  Write-Output "Installing WiX Toolset"
                   choco install --no-progress -y wixtoolset
-                  Write-Output "Reloading PATH"
-                  $env:Path = [System.Environment]::GetEnvironmentVariable("Path","Machine") + ";" + [System.Environment]::GetEnvironmentVariable("Path","User")
-                  [Environment]::SetEnvironmentVariable("PATH", $env:Path + "C:\Program Files (x86)\WiX Toolset v3.11\bin;", "Machine")
-                  $env:Path = [System.Environment]::GetEnvironmentVariable("Path","Machine") + ";" + [System.Environment]::GetEnvironmentVariable("Path","User")
-                  npm config set node-gyp "$CIRCLE_WORKING_DIRECTORY\node_modules\node-gyp\bin\node-gyp.js"
-
-                  yarn lint
-                  yarn build
-                  yarn test:slow
+                  echo 'export PATH=$PATH:"/C/Program Files (x86)/WiX Toolset v3.11/bin"' >> "$BASH_ENV"
+                  cd 'C:\Program Files\nodejs\node_modules\npm\node_modules\@npmcli\run-script'
+                  npm install node-gyp@latest
```

Comment:
Should we pin this to a specific version? node-gyp doesn't release terribly often, but it's often enough that we may want to watch for it.

Reply:
[7e9d37f](https://github.com/electron/forge/pull/3319/commits/7e9d37f64197d3782165eab3c81f661b48e4fcea)